sql/core/src/test/scala/org/apache/spark/sql/connector/WriteDistributionAndOrderingSuite.scala

```diff
@@ -778,8 +778,7 @@ class WriteDistributionAndOrderingSuite
 
     sparkContext.listenerBus.waitUntilEmpty()
 
-    assert(executedPlan.isInstanceOf[CommandResultExec])
-    executedPlan.asInstanceOf[CommandResultExec].commandPhysicalPlan match {
+    executedPlan match {
```

Comment:
Why we need this change?

Reply:
Oh. I got it.

Reply:
I fixed https://github.com/apache/spark/pull/32919/files#diff-23a90192be7d7c83780c61292bc061b2eddb79de6cd94d8234dddce781f60c1eL195

So I reverted these changes from your previous commit.